Jenkinsfile

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env groovy
 
+library 'devops-tools-jenkins'
+
+def dockerRegistry = libraryResource('dockerregistry').trim()
+def dockerImage = "${dockerRegistry}/bbc-news/elixir-centos7:1.11.3"
```

Comment:
Do we have PRs for all other elixir applications such as Belfrage, Mozart Fetcher, otherwise they will use the latest image?
> https://github.com/bbc/belfrage/blob/master/Jenkinsfile#L6

Reply:
I plan to test this one before merging and getting the others reviewed, you can keep up with the list of which I have done so far here: https://jira.dev.bbc.co.uk/browse/RESFRAME-4167

The image with the latest tag is the older version (elixir 1.8.2 erlang 21.1.1) this should not break anything for the two jenkins files which use the elixir-centos7 (belfrage and belfrage smoke-tests)